model_zoo/census_model_sqlflow/wide_and_deep/wide_deep_subclass_keras.py

```diff
@@ -0,0 +1,244 @@
+import tensorflow as tf
+
+from elasticdl.python.elasticdl.callbacks import LearningRateScheduler
```

Comment:
Can we have an example that does not import `elasticdl` so that we can test run this python script on any environment that does not have elasticdl installation

Reply:
Good idea, I'll remove "import elasticdl" in this code example.

Reply:
For SparseEmbedding layer, we will move it from elasticdl package to elasticdl_preprocessing package in the next PR.